packages/jest-config/src/index.ts

```diff
@@ -121,6 +121,7 @@ const groupOptions = (
     replname: options.replname,
     reporters: options.reporters,
     rootDir: options.rootDir,
+    runInBand: options.runInBand,
```

Comment:
Not sure if this should be added to config object. I have a feeling it was not here intentionally.

Just an idea, I did not check if this would work. Perhaps `getMaxWorkers()` should return `0` instead of `1` (sort of: "no workers, please") if `runInBand` is passed?

https://github.com/facebook/jest/blob/15af9b3b6f53a4f2f2e39d328eb6ef9f46ba91c4/packages/jest-config/src/getMaxWorkers.ts#L27-L28